<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Summer School of Science</title>

    <!-- Bootstrap Core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom CSS -->
    <link href="css/sss.css" rel="stylesheet">

    <!-- Custom Fonts -->
    <link href="font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
    
    <script src="js/jquery-1.6.1.min.js"></script>
	<script src="js/bootstrap.min.js"></script>
	<script src="js/init.js"></script>

    <!-- Google Analytics -->
    <!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-64967324-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-64967324-1');
</script>

</head>

<body id="page-top">

    <!-- Navigation -->
    <nav class="navbar navbar-custom navbar-fixed-top top-nav-collapse" role="navigation">
        <div class="container">
            <div class="navbar-header">
                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-main-collapse">
                    <i class="fa fa-bars"></i>
                </button>
                <a class="navbar-brand page-scroll" href="index.html">
                <span class="light" class="light" style="font-family: Arial;">S<sup style="font-family: Arial;">3</sup>/S<sup style="font-family: Arial;">3</sup>++</span>
                </a>
            </div>

            <!-- Collect the nav links, forms, and other content for toggling -->
            <div class="collapse navbar-collapse navbar-right navbar-main-collapse">
                <ul class="nav navbar-nav">
                    <!-- Hidden li included to remove active class from about link when scrolled up past about section -->
                    <li class="hidden">
                        <a href="#page-top"></a>
                    </li>
                    <li>
                        <a class="page-scroll nav" href="index.html">About</a>
                    </li>
                    <li class="dropdown">
                        <a href="#" data-toggle="dropdown" class="dropdown-toggle nav">Program <b class="caret"></b></a>
                        <ul class="dropdown-menu">
                           <li><a href="s3_2021.html">S<sup>3</sup> 2021</a></li>
                            <li><a href="s3pp_2021.html">S<sup>3</sup>++ 2021</a></li>
                            <li><a href="archive.html">Archive</a></li>
                            <li><a href="daytoday.html">Day-to-day activities</a></li>
                            <li><a href="index.html#practical">Practical info</a></li>
                        </ul>
                    </li>
                    <li>
                        <a class="page-scroll nav" href="newsfeed.html">News</a>
                    </li>
                    <li>
                        <a class="page-scroll nav" href="people.html">People</a>
                    </li>
                    <li>
                        <a class="page-scroll nav" href="apply.html">Apply</a>
                    </li>
                    <li>
                        <a class="page-scroll nav" href="sponsors.html">Sponsors</a>
                    </li>
                    <li>
                        <a class="page-scroll nav" href="press.html">Press</a>
                    </li>
                    <li>
                        <a class="page-scroll nav" href="faq.html">FAQ</a>
                    </li>
                    <li>
                        <a class="page-scroll nav" href="index.html#contact">Contact</a>
                    </li>
                </ul>
            </div>
            <!-- /.navbar-collapse -->
        </div>
        <!-- /.container -->
    </nav>

    
    <!-- About Section11 -->
    <section id="about" class="content-section text-center">
        <div class="break-section2">
            <div class="container">
            	<div class="col-lg-8 col-lg-offset-2">
                    <h2>S<sup>3</sup>++ 2021</h2>
                    <p > Požega, August 5<sup>th</sup> - August 15<sup>th</sup></p>
            
                    <a href="#projects" class="page-scroll btn btn-default btn-lg">Projects</a>
                    <a href="#workshops" class="page-scroll btn btn-default btn-lg">Workshops</a>
                    <a href="#lectures" class="page-scroll btn btn-default btn-lg">Lectures</a>

                    
                </div>
            </div>
        </div>    
    </section>






    <section id="projects" class="content-section text-center">
        <div class="break-section2">
            <div class="container">
                <div class="col-lg-8 col-lg-offset-2">
                    <h2>Projects</h2>
                    <div class="lineb"></div>

                    
                    <p class="media">
			 <h3><strong><a name="p1" id="p1"></a>Explaining source code to computers: helping them help us</strong><br/></h3>

                        <p class="project"><img src="img/2020/Marko_project.jpg"  height="170" hspace="9" align="right" />People use programming languages to talk to computers---mostly to boss them around and give them repetitive tasks to execute. However, computers cannot execute instructions written in high-level languages like C++, Java or Python directly---their processors can only execute very simple machine code. To help them out, we use a typically rather complex program called a compiler which reads high-level source code and translates it into low-level machine code. While enormous compilers like gcc are monumental achievements of the community with millions of lines of code and hundreds of contributors, smaller compilers or interpreters for simple, but useful languages can be written by a handful of enthusiastic people during a really cool summer school.

                        </p>

                        <p class="project"> In this project, we are going to investigate a wide variety of programming languages as use cases for some of the most important program compilation and interpretation stages, including lexing, parsing and evaluation. After an initial short presentation of Python and relevant topics in theoretical Computer Science, reading and programming assignments will be handed out dependent on complexity and team size. The general plan is to code up interpreters for simple languages with very different purposes, including, time permitting, our own custom-built languages. The student will learn how to read and/or write a specification for a language as well as syntactically and semantically analyse source code written in the language.
                        </p>
                    
                    <div class="media">
                        <div class="media-left">
                            <a href="">
                                <img class="media-object" src="img/2020/Marko.jpg" width="145" hspace="9" vspace="14" align="left" />
                            </a>
                        </div>
                        <div class="media-body">
                            <p class="project"><strong>Marko Horvat</strong><br />
                                Faculty of Science, University of Zagreb, Croatia<br/><br/>

                         Marko is currently a postdoc at the Department of Mathematics, Faculty of Science, University of Zagreb, Croatia. His research and teaching tends to be in theoretical Computer Science, such as formal verification in Information Security.  He studied Maths---mostly Logic---in Zagreb, started his PhD in Computer Science at ETH Zurich, completed it at the University of Oxford and he did a postdoc at MPI-SWS in Kaiserslautern, Germany. Nowadays he can be seen thinking about computable topology. In his free time, Marko enjoys video and board games, post-punk and metal music, and coffee.

                            </p>
                        </div>
                    </div>
			
			 <h3><strong><a name="p2" id="p2"></a>To fold or not to fold? RNA structure and function</strong><br/></h3>

                        <p class="project"><img src="img/2021/Lukas_project.jpg"  height="170" hspace="9" align="right" />
According to the central dogma of molecular biology, RNA is often considered to be “just” the messenger of the genetic information between DNA to proteins. But RNA is not just some “wiggly” noodle floating in the cytoplasm. It can do much more, it also supplies the amino acids for translation (tRNA), catalyzes chemical reactions (rRNA, ribozymes) as well as regulates various pathways in the cell (miRNA, siRNA, sncRNA). In fact, RNA can do anything! And it is not only the sequence, but similarly to proteins, also the structure that matters. 


                        </p>

                        <p class="project">In this project, we will focus on the RNA structure and its functional 
consequences. First, we are going to employ several RNA structure 
prediction algorithms in order to verify their accuracy by comparing 
with the known RNA elements from the literature. Once we have done that 
and gained some experience, we will be able to look for specific 
elements within various RNA sequences, including human genes, tRNAs or viral genomes. After all of this, you would have learned what is the connection between RNA structure and its function with a little bit of 
coding as well. </p>
                    
                    <div class="media">
                        <div class="media-left">
                            <a href="">
                                <img class="media-object" src="img/2020/Lukas.jpg" width="145" hspace="9" vspace="14" align="left" />
                            </a>
                        </div>
                        <div class="media-body">
                            <p class="project"><strong>Lukáš Pekárek</strong><br />
                                Julius Maximilian University of Würzburg - Helmholtz Institute for RNA-based Infection research Infection biology, Germany<br/><br/>

Lukáš is a PhD student at the University of Würzburg, Germany. He works as a doctoral researcher at the Helmholtz Institute for RNA-based Infection research in Neva Caliskan’s group focused on Non-canonical translation events. He is currently working on the use of single-molecule techniques to unveil how viruses hijack our cells. Before that, he did his Bachelor and Master at the University of Chemistry and Technology in Prague. Outside the lab, he likes white-water canoeing and camping.
                            </p>
                        </div>
                    </div>

			 <h3><strong><a name="p3" id="p3"></a>To fold or not to fold? RNA structure and function</strong><br/></h3>

                        <p class="project"><img src="img/2021/Paula_project.jpg"  height="170" hspace="9" align="right" />
Our genomes are constantly suffering damage from factors outside the cell, such as environmental factors, or from harmful molecules produced by normal cell metabolism. Cells are often not able to repair the DNA damage caused by different factors which may lead to the development of cancer. So far, scientists have identified multiple distinct fingerprints of damage in the DNA, known as a mutational signature, that are left by specific causes of cancer, like ultraviolet light and tobacco use. These unique mutational signatures help us understand how certain cancers have developed and how they could have been prevented. Also, mutational signatures are starting to show their importance in providing a better diagnosis and more effective therapy for a patient.

                        </p>

                        <p class="project">
			In this project, we are going to analyze mutational signatures in cancer patients to identify which specific processes are responsible for the development of cancer. Our main tool in this project is going to be the programming environment R. Besides learning a bit of basic R, we will learn how to obtain publicly available genomic data, analyze mutations and detect specific mutational signatures using machine learning algorithms. Moreover, we will analyze the association of our mutational signatures with distinct clinical outcomes of patients. 
			</p>
                    
                    <div class="media">
                        <div class="media-left">
                            <a href="">
                                <img class="media-object" src="img/2020/Paula.jpg" width="145" hspace="9" vspace="14" align="left" />
                            </a>
                        </div>
                        <div class="media-body">
                            <p class="project"><strong>Paula Štancl</strong><br />
                              Faculty of Science, University of Zagreb, Croatia<br/><br/>

Paula is currently a PhD Student at the Faculty of Science, Croatia. As a part of the Bioinformatics group at her faculty, she is interested in the mutational landscape of cancers and its association to different chromatin features. During her high school years, she participated at S3, and later went on as a Swaphop leader in 2016. In her free time, she likes to play video games and watch movies.                             </p>
                        </div>
                    </div>

<!--
                   



                     <h3><strong><a name="p1" id="p1"></a>Science, medicine, and the future: RNA sequencing </strong><br/></h3>
                        <p class="project"><img src="img/2020/Ivana_project.jpeg" width="260"  hspace="9" align="right" />Revolution of modern biology has started more then 20 years ago by introducing the sequencing of any genome we could possibly imagine. Furthermore, the development of single-cell RNA sequencing (scRNA-seq) has enabled access to information about gene expression in individual cells and insights into new biological areas. This not only lead us to the better understanding of human diseases, but also enabled us to raise the importance of tailoring a disease therapy to each individual. The method has proven to be especially important for understanding serious illnesses like brain cancer.
 
                        </p>

                        <p class="project">In this project we are going to apply well know machine learning algorithms to tailor personalized approach in treating patients with glioblastoma, the most aggressive brain tumor. To accomplish that, we are going to use a popular program language R to develop a code that will be applied to analyze a real scRNA seq dataset. We are going to learn how to investigate the quality of any given dataset and how to preprocess the data before making any conclusions of biological value. Furthermore, we will learn what is cell clustering, how to perform one and how to identify found cells in patient sample based on “gene markers”. In the end we will build a code that will tell us which specific cells react to patient tumor, which is the key information clinician use to create personalized vaccines against patient’s tumor. 
                        </p>

                        <p class="project">This project will introduce you to the modern and rapidly expanding pipeline of treating patient’s cancer using only vaccines. More important, it will show you the importance of bioinformatics and processing large datasets without which we cannot imagine modern science, medicine and future. </p>
                    
                    <div class="media">
                        <div class="media-left">
                            <a href="">
                                <img class="media-object" src="img/2020/Ivana.jpeg" width="135" hspace="9" vspace="14" align="left" />
                            </a>
                        </div>
                        <div class="media-body">
                            <p class="project"><strong>Ivana Osredek</strong><br />
                                University of Leicester, UK<br/><br/>

                            Ivana is currently a third-year student at the University of Leicester, UK. This year she is doing Erasmus + internship at the German Cancer Research Centre. Her work involves identification of tumor-reactive T cell receptors using single-cell sequencing and bioinformatics tools, in order to create a personalized treatment for patients suffering from brain cancer. In her free time Ivana is an active trail long distance runner, currently preparing for a marathon. 

                            </p>
                        </div>
                    </div>



                     


                    <div class="line2"></div>

                     <h3><strong><a name="p1" id="p1"></a>Explaining source code to computers: helping them help us</strong><br/></h3>

                        <p class="project"><img src="img/2020/Marko_project.jpg"  height="170" hspace="9" align="right" />People use programming languages to talk to computers---mostly to boss them around and give them repetitive tasks to execute. However, computers cannot execute instructions written in high-level languages like C++, Java or Python directly---their processors can only execute very simple machine code. To help them out, we use a typically rather complex program called a compiler which reads high-level source code and translates it into low-level machine code. While enormous compilers like gcc are monumental achievements of the community with millions of lines of code and hundreds of contributors, smaller compilers or interpreters for simple, but useful languages can be written by a handful of enthusiastic people during a really cool summer school.

                        </p>

                        <p class="project"> In this project, we are going to investigate a wide variety of programming languages as use cases for some of the most important program compilation and interpretation stages, including lexing, parsing and evaluation. After an initial short presentation of Python and relevant topics in theoretical Computer Science, reading and programming assignments will be handed out dependent on complexity and team size. The general plan is to code up interpreters for simple languages with very different purposes, including, time permitting, our own custom-built languages. The student will learn how to read and/or write a specification for a language as well as syntactically and semantically analyse source code written in the language.
                        </p>
                    
                    <div class="media">
                        <div class="media-left">
                            <a href="">
                                <img class="media-object" src="img/2020/Marko.jpg" width="145" hspace="9" vspace="14" align="left" />
                            </a>
                        </div>
                        <div class="media-body">
                            <p class="project"><strong>Marko Horvat</strong><br />
                                Faculty of Science, University of Zagreb, Croatia<br/><br/>

                         Marko is currently a postdoc at the Department of Mathematics, Faculty of Science, University of Zagreb, Croatia. His research and teaching tends to be in theoretical Computer Science, such as formal verification in Information Security.  He studied Maths---mostly Logic---in Zagreb, started his PhD in Computer Science at ETH Zurich, completed it at the University of Oxford and he did a postdoc at MPI-SWS in Kaiserslautern, Germany. Nowadays he can be seen thinking about computable topology. In his free time, Marko enjoys video and board games, post-punk and metal music, and coffee.

                            </p>
                        </div>
                    </div>

                    <div class="line2"></div>



                     <h3><strong><a name="p1" id="p1"></a>Fantastic proteins and how to produce them  </strong><br/></h3>
                        <p class="project"><img src="img/2020/Lukas_project.JPG" width="270"  hspace="9" align="right" />
                        Late advances of molecular biology have changed our lives. Less than 70 years ago, no one knew how the genetic information is transferred from generation to generation. Since then a huge progress has been achieved. Advances in cloning, sequencing and other techniques culminated in the publication of first human genome sequence by the early 2000s. Among other discoveries, two big breakthroughs are worth mentioning. The recombinant protein production – a technique that uses microorganisms, such as bacteria or yeast, to produce specific proteins (such as human insulin) in a large scale. Introduction of the recombinant protein production transformed the pharmaceutical industry and allowed us to treat previously fatal diseases such as diabetes. The other great advancement is the polymerase chain reaction (PCR). PCR allows (not only) scientist to easily and precisely multiply any segment of DNA. Today, PCR is a routine method used in almost any biochemical lab. Not only it is used world-wide in the field of science, PCR is also an essential for assigning DNA origin in criminology as well as paternity tests.
 
                        </p>

                        <p class="project">In this project, we will combine the two above mentioned molecular biology techniques – Recombinant protein production and PCR. First, we will grow E. coli bacterial cells and use them to produce recombinant protein – a thermostable DNA polymerase derived from thermophilic Archea Pyrococcus furiosus. The very similar procedure is used, in much bigger scale, in pharmaceutical industry. Subsequently, we will purify the protein. To verify successful production, we will use our “homemade” DNA polymerase as the key component in the PCR reaction and amplify various DNA molecules. 


                        </p>
                    
                    <div class="media">
                        <div class="media-left">
                            <a href="">
                                <img class="media-object" src="img/2020/Lukas.jpg" width="135" hspace="9" vspace="14" align="left" />
                            </a>
                        </div>
                        <div class="media-body">
                            <p class="project"><strong>Lukáš Pekárek</strong><br />
                                Julius Maximilian University of Würzburg, Germany<br/><br/>

                            Lukáš is a PhD student at the University of Würzburg, Germany. He works as a doctoral researcher at the Helmholtz Institute for RNA-based Infection research in Neva Caliskan’s group focused on Non-canonical translation events. He is currently working on the use of single-molecule techniques to unveil the how viruses hijack our cells. Before that, he did his Bachelor and Master at the University of Chemistry and Technology in Prague. Outside the lab, he likes white-water canoeing and camping.
                            </p>
                        </div>
                    </div>

                    <div class="line2"></div>




                                         <h3><strong><a name="p1" id="p1"></a>Backyard science: construct your own particle detector</strong>
                     </h3>
                        <p class="project"><img src="img/2020/Anton_project.jpg" width="280" hspace="9" align="right" />Throughout all of mankind’s history, one of the central questions was the structure of matter and reality. This issue was mostly unresolved until the 20th century, when the Standard model of particle physics answered a lot of the questions, but also opened more sophisticated challenges and opportunities. While knowledge about the building blocks of the Universe is its own reward, humanity as a whole also profits from direct and indirect discoveries, often in unforeseen ways. For example, the result of a need for networking and sharing data between scientists at CERN is the modern Internet. Also, modern particle accelerators are extremely large and involve thousands of people working together in various ways, including theorists giving predictions, computer scientists developing algorithms and software, and technicians and experimental physicists working on the detector itself.

                        </p>

                        <p class="project">The goal of this project is to construct a particle detector from easily obtainable materials, like copper wire and fish tanks, and also try to make some rudimentary measurements. Since it’s impractical to build a big accelerator in a few days, the detector will mainly observe naturally accelerated particles found in cosmic rays. Detections will be filmed with smartphones and processed using computers and free software. At the same time, in order to understand our results, we’re going to learn as much theory as time permits, which includes some underlying math and physics, types of particles and interactions, and methods of data analysis.


                        </p>
                    
                    <div class="media">
                        <div class="media-left">
                            <a href="">
                                <img class="media-object" src="img/2020/Anton.jpg" width="135" hspace="9" vspace="14" align="left" />
                            </a>
                        </div>
                        <div class="media-body">
                            <p class="project"><strong>Anton Perkov</strong><br />
                                Faculty of Science, University of Zagreb, Croatia<br/><br/>

                            
                                Anton is currently a PhD student at the Department of Physics in Zagreb, Croatia, where he previously graduated. He works on the physics of high-energy hadron scatterings and phenomenology of quantum chromodynamics, the theory of the strong force. He was a participant at S3 during high school, and returns for the first time as a project leader. In his free time he practices Japanese swordsmanship, listens to various audiobooks, and plays the piano.


                            </p>
                        </div>
                    </div>


                    
                    

                    <div class="line2"></div>




                                         <h3><strong><a name="p1" id="p1"></a>Dark Patterns -  How to create addictive technology</strong>
                     </h3>
                        <p class="project"><img src="img/2020/JuliaHT_project.png" width="260" hspace="9" align="right" />Ever open an app and spend much longer on it than intended? Millenials spend an average of 3.2 hours per day on their phones. That’s a full day each week! Most digital applications like Instagram, YouTube,  Snapchat and WhatsApp are specifically designed to be highly addictive. Designers use consumer psychology to understand, manipulate and control people’s behaviour. Consumer psychology is a field which examines why we buy products and services and how we use them. Learning about this field allows you to become more aware of the ways in which products and services you encounter regularly influence you. 

                        </p>

                        <p class="project">In this project, through the lens of consumer psychology, we will examine the many techniques and tricks or “dark patterns” that designers use to keep people engaged with their product or app as long as possible (typically much longer than intended). We will do fun activities and create and carry out experiments designed to convey relevant psychological concepts. Then, ultimately, you will dive in and design an app prototype using all the features and techniques we have learnt. By the end of the project, you will be an expert on the methods being used to get and keep you hooked on digital products and will be better equipped to resist these strategies in the future. 


                        </p>
                    
                    <div class="media">
                        <div class="media-left">
                            <a href="">
                                <img class="media-object" src="img/2020/JuliaHT.png" width="135" hspace="9" vspace="14" align="left" />
                            </a>
                        </div>
                        <div class="media-body">
                            <p class="project"><strong>Julia Hamblin-Trué</strong><br />
                                CODE University of Applied Sciences, Germany<br/><br/>

                            
                                Julia is studying at CODE University in Berlin and majoring in Product Management, which focuses on guiding every step of a product’s development. One of her favourite courses was Consumer Psychology and she went on to teach this course to fellow students. She is very interested in the ethical considerations of technology in society. She enjoys acting, singing, playing instruments, cooking and running in nature. Julia was part of the past three S3++ Camps as a participant and a co-lead. 



                            </p>
                        </div>
                    </div>


                    
                    

                    <div class="line2"></div>

                        

                                     <h3><strong><a name="p1" id="p1"></a>Fun and games and mathematics: applying game theory to real-life decision making</strong><br/></h3>
                        <p class="project"><img src="img/2020/Dario_project.jpg" width="240"  hspace="9" align="right" />Game theory is a mathematical discipline that sounds like all that it studies is fun and games, but quite the contrary: it’s the science of logical decision making in humans, animals and computers. We’ve seen game theorists making significant contributions to different fields such as biology and economics and even winning the Nobel Prize in Economic Sciences. As it studies the strategic interaction among decision-makers, it’s been recognized as an important tool in many fields.
 
                        </p>

                        <p class="project">In this project, we’ll provide an introduction to game theory, covering the basic game types, representations of games, its applied uses (which go as far as to affect the pricing of products in retail) and its most important achievements, problems and results. Furthermore, we’ll look at some games that have been thoroughly analyzed by game theorists, such as the Prisoner’s dilemma, the Bargaining problem and the Dictator game, as well as some mathematically analyzed popular games such as Set, and what does formal game analysis consist of. Then, having in mind what we’ve learned, we’ll design our own games, let others play it and take a note of their strategies. This will allow us to both find the best strategies for our games (by analyzing them mathematically) and the most common strategies used by our players, which will finally let us analyze the best/average strategy relationship. The students will be introduced to game theory and learn game design and analysis, as well as some basic statistics and general mathematical methods and how to apply mathematical reasoning to seemingly not-very-mathematical problems.



                        </p>
                    
                    <div class="media">
                        <div class="media-left">
                            <a href="">
                                <img class="media-object" src="img/2020/Dario.jpg" width="135" hspace="9" vspace="14" align="left" />
                            </a>
                        </div>
                        <div class="media-body">
                            <p class="project"><strong>Dario Ćosić</strong><br />
                                Faculty of Science, University of Zagreb, Croatia<br/><br/>

                            Dario is an undergraduate student of mathematics at the Faculty of Science in Zagreb. His interests lie in number theory, game theory, real analysis, discreet mathematics and combinatorics. In his spare time, he likes to play the guitar, watch a good movie or take a swim in his hometown of Zadar. 
                            </p>
                        </div>
                    </div>



                     


                    <div class="line2"></div>




                -->



             
              



                   






                    




                    
                    </p>
                </div>
            </div> 
        </div>
    </section>


    
    <section id="workshops" class="content-section text-center">
        <div class="break-section">
            <div class="container">
                <div class="col-lg-8 col-lg-offset-2">
                    <h2>Workshops <br/>
</h2>
                    <div class="lineb"></div>
                    
                    <div id="swapshops">

                         
                        <p><i>TBA</i></p>
                                   

                        <div class="line2"></div>




                        
                        
                    
                      

                    </div>
                </div>
            </div>
        </div>
    </section>




    <section id="lectures" class="content-section text-center">
        <div class="break-section2">
            <div class="container">
        
                <div class="col-lg-8 col-lg-offset-2">
                    <h2>Lectures</h2>
                    <div class="lineb"></div>
                    <p><i>TBA</i></p>

                    <!--
                
                    <h3>Lecture schedule</h3>


                    <p class="main2">
                        August 7th, 2020 <br/>
                        <strong>Jana Pisk</strong> (University of Zagreb, Croatia): <i><a href="files/lectures/JanaPiskSummary.pdf"><strong>Can molecules run?</strong></a></i>
                    </p>

                    <p class="main2">
                        August 8th, 2020 <br/>
                        <strong>Letizia Jaccheri</strong> (Norwegian University of Science and Technology, Norway): <i><a href="files/lectures/LetiziaJaccheri_Summary.pdf"><strong>From software through art to social entrepreneurship</i></strong></a>
                    </p>

                     <p class="main2">
                        Date TBA <br/>
                        <strong>Leonardo Pierobon</strong> (ETH Zurich, Switzerland): <i><a href="files/lectures/LeonardoPierobon_TalkSummary.pdf"><strong>Nanostructuring permanent magnets for green technologies</i></strong></a>
                    </p>




                    <div class="line2"></div>
                    <h3>About the lecturers</h3>




                             <div id="lecture3">
                    <div class="media">
                            <div class="media-left">
                                <a href="">
                                    <img class="media-object" src="img/2020/Lecturers/Jana_Pisk.png" alt="" width="135" hspace="9" vspace="7" align="left" />
                                </a>
                            </div>
                            <div class="media-body">
                                <p class="project"><strong>Jana Pisk</strong><br />
                                    University of Zagreb, Croatia<br/><br/>

                               Jana is Assistant Professor at the Faculty of Science, Department of Chemistry. As a Ph.D. she researched <strong>moleybdenum coordination complexes</strong> and theirs application for <strong>epoxidation reactions</strong>. After the Ph.D. she continued the reserach of molecular and heterogeneous <strong>catalysts</strong> and their application in different catalytic processes. In her free time, Jana enjoys coffees and travelling. Japan is her prefered destination at the moment.


                                </p>
                            </div>
                        </div>
                    </div>


                   
                    <div id="lecture1">
                    <div class="media">
                            <div class="media-left">
                                <a href="">
                                    <img class="media-object" src="img/2020/Lecturers/Letizia_Jaccheri.jpg" alt="" width="135" hspace="9" vspace="7" align="left" />
                                </a>
                            </div>
                            <div class="media-body">
                                <p class="project"><strong>Letizia Jaccheri</strong><br />
                                    Norwegian University of Science and Technology, Norway<br/><br/>

                               Letizia Jaccheri is Professor at the Department of Computer Science of the Norwegian University of Science and Technology. Her research is on <strong>software engineering, entertainment computing, computational creativity, and ICT-enabled social innovation</strong>. Dr. Jaccheri is the Norwegian representative and Vice President of IFIP TC14 on Entertainment Computing. She has published more than 200 papers in International conferences and journals. From 2015 to April 2018 she was independent <strong>director of Reply S.p.A</strong>, an IT company with 6,000 employees world wide. Letizia Jaccheri is passionate about dissemination of computer science and research to the general public and to contribute to recruit female students to computer science and research. 


                                </p>
                            </div>
                        </div>
                    </div>





                    <div id="lecture2">
                    <div class="media">
                            <div class="media-left">
                                <a href="">
                                    <img class="media-object" src="img/2020/Lecturers/Leonardo_Pierobon.jpg" alt="" width="135" hspace="9" vspace="7" align="left" />
                                </a>
                            </div>
                            <div class="media-body">
                                <p class="project"><strong>Leonardo Pierobon</strong><br />
                                    ETH Zurich, Switzerland<br/><br/>

                              The vastness of the universe fascinated Leonardo as a child, who fell in love with physics and astronomy through learning about black holes. As he grew older, Leonardo’s curiosity grew bigger, and he decided to study Natural Sciences at the University of Cambridge, where his interests shifted from <strong>cosmic expanses</strong> to <strong>nanoscopic scales</strong>. This is why he is now doing his PhD in <strong>nanomagnetism</strong> at ETH Zurich, combining  experiments in electron microscopy and micromagnetic simulations to make better magnetic materials. Leonardo joined the Summer School of Science in 2016 as a project leader and returned as an organizer of the S3++ in 2018 and 2019. Leonardo strongly embraces the diversity and tolerance of international environments and looks forward to the Summer School of Science where everyone is free to be who they are.


                                </p>
                            </div>
                        </div>
                    </div>


                -->
                    

                </div>
            </div>
        </div>
    </section>






    <!-- Footer -->
    <footer>
    <hr> 
        <div class="container text-center">
            <p><font size="2">Copyright &copy;  Society for Out-of-Frame Education. All Rights Reserved.</font></p>
                <p><font size="2">Webdesign: Sebastijan Dumančić, Igor Jukić, Lucija Šilić, Jelena Tica. <br> General text: Sebastijan Dumančić, Petra Krznar, Branimir Lukić, Martina Mijušković, Nikolina Šoštarić </font></p>
        </div>
    </footer>

    <!-- jQuery -->
    <script src="js/jquery.js"></script>

    <!-- Bootstrap Core JavaScript -->
    <script src="js/bootstrap.min.js"></script>

    <!-- Plugin JavaScript -->
    <script src="js/jquery.easing.min.js"></script>

    <!-- Custom Theme JavaScript -->
    <script src="js/single.sss.js"></script>

    

</body>

</html>
